Hunter/Assets/Scripts/View/AnimalGizmos.cs

```diff
@@ -0,0 +1,13 @@
+using UnityEngine;
+
+public class AnimalGizmos : MonoBehaviour
+{
+    [SerializeField]
+    private float _bodySeekRadius;
+
+    private void OnDrawGizmos()
+    {
+        Gizmos.color = Color.green;
+        Gizmos.DrawWireSphere(transform.position, _bodySeekRadius);
+    }
+}
```

Comment:
Firstly - I've added new script. it shows Gizmos - usefull shapes, which reveals something. In My case - zone of vision. They are visible only in Scene